synapse/rest/media/v1/download_resource.py

```diff
@@ -45,7 +45,13 @@ def render_GET(self, request):
     @request_handler()
     @defer.inlineCallbacks
     def _async_render_GET(self, request):
-        request.setHeader("Content-Security-Policy", "sandbox")
+        request.setHeader(
+            "Content-Security-Policy",
+            "default-src none;"
+            " plugin-types application/pdf;"
+            " style-src 'unsafe-inline';"
```

Comment:
The CSP is saying that it can't run anything (`default-src none` implies `script-src 'none'`), except inline style sheets. However, turns out that PDF documents can run certain bits of javascript, but having looked online it feels that its quite limited on what javascript in can run. In particular it can't do HTTP requests and the like, so we should be OK.